package/package.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
Is this package folder's files needed?
is it created by error?

Reply:
ah, yeah its by mistake, i will remove it